src/savegame_json.cpp

```diff
@@ -1513,6 +1514,7 @@ void vehicle::serialize(JsonOut &json) const
     json.member( "camera_on", camera_on );
     json.member( "dome_lights_on", dome_lights_on );
     json.member( "aisle_lights_on", aisle_lights_on );
+    json.member( "has_atomic_lights", has_atomic_lights );
```

Comment:
You don't need to export and load this, when a vehicle has been loaded, `refresh` is automatically called, which will set this value. See savegame_json.cpp, `vehicle::deserialize`